src/state/farms/fetchFarms.ts

```diff
@@ -1,15 +1,47 @@
 import { SerializedFarmConfig } from 'config/constants/types'
-import fetchFarm from './fetchFarm'
```

Comment:
I think `fetchFarm.ts` is not used anymore and can be deleted?

Reply:
Done